test/helpers/sequence.ts

```diff
@@ -0,0 +1,64 @@
+import Listr from 'listr';
+
+import { TestRunner } from './test-runner';
+import { log } from './utils';
+
+export async function runTestIntegrationSequence(version: string) {
+  const runner = new TestRunner(version);
+
+  if (process.env.DEBUG) {
```

Comment:
I don't grok this conditional, isn't this _exactly_ what `listr` is doing just below?

Reply:
mostly, yeah. I want exploring listr, because it looks nice. But it also doesn't provide any visibility into commands when things go wrong. :(

Reply:
listr removed. :tada: